install-panel.sh

```diff
@@ -136,35 +136,16 @@ required_input() {
 }
 
 password_input() {
-  local  __resultvar=$1
-  local  result=''
-
-  while [ -z "$result" ]; do
-    echo -n "* ${2}"
-
-    # modified from https://stackoverflow.com/a/22940001
-    while IFS= read -r -s -n1 char; do
-      [[ -z $char ]] && { printf '\n'; break; } # ENTER pressed; output \n and break.
-      if [[ $char == $'\x7f' ]]; then # backspace was pressed
-          # Only if variable is not empty
-          if [ -n "$result" ]; then
-            # Remove last char from output variable.
-            [[ -n $result ]] && result=${result%?}
-            # Erase '*' to the left.
-            printf '\b \b' 
-          fi
-      else
-        # Add typed char to output variable.
-        result+=$char
-        # Print '*' in its stead.
-        printf '*'
-      fi
+  local result=$1
+  echo -e "Please enter the password that is used throughout the install. (Use something strong)"
+  while [[ -z "$passOriginal" || ! "$passOriginal" == "$passConfirm" ]]
```

Comment:
This project does not use camel case. While the rest of the project does have a few inconsistencies, I try to adhere to generally accepted naming conventions. [Google's Style Guide](https://google.github.io/styleguide/shellguide.html#s7-naming-conventions) has a few tips regarding variable naming conventions.

Reply:
No worries ill change this